### DISCUSSION TITLE
alternative.

### DISCUSSION THREAD

**Discussion by gooseisgoose** [3kh0/ext-remover#941] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is there a alternative for Pollen. i don't have a chromebook.

**Comment by AshtonDavies** [comment]
If you have a Windows computer, I believe there is a built-in policy editor somewhere.

**Reply by gooseisgoose** [reply]
ok i'll look thanks. and Damn you fast at replying

**Reply by gooseisgoose** [reply]
it wont let me it says i don't have the permissions to do that. i think i went to the right place. or maybe i'm just screwed.

**Comment by AshtonDavies** [comment, marked as answer]
Probably are

**Reply by gooseisgoose** [reply]
fuck
